### DISCUSSION TITLE
The difference between model and schema

### DISCUSSION THREAD

**Discussion by jason0376** [hexatrail/fastapi-example#2] [Q&A] [answered]
what is the difference between model and schema in fastapi structure?
why do we use schema in the FastAPI?

**Comment by felipeserrano05** [comment, marked as answer]
In FastAPI:

- Model: Represents the database structure, often defined with an ORM like SQLAlchemy. It reflects how data is stored.
- Schema: Defines the request and response structure, often using Pydantic. It validates and controls the data format for API input/output.
Why Use Schema:
Schemas ensure data validation, serialization, and control over which fields are exposed to the client, keeping database models separate from API logic.
